### PR TITLE
BIP 86: Key Derivation for Single Key P2TR Outputs

### PR DESCRIPTION
**Pull request by achow101**
This BIP specifies a suggested derivation path to use for single key P2TR outputs, following BIP 43/44.

Test vectors and the actual derivation path value for the purpose will be set after receiving a BIP number.

### CODE REVIEW

**Comment by dr-orlovsky** [comment]
Have you seen by earlier ideas around more generic support for both single- and multisig wallets for P2TR in bitcoin-dev?

https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-February/018381.html

**Comment by achow101** [comment]
> Have you seen by earlier ideas around more generic support for both single- and multisig wallets for P2TR in bitcoin-dev?

Yes, but I think more complicated derivation schemes can come later. Using the system we already have is the path of least resistance and will allow for faster adoption of Taproot in current wallets.

File: bip-taproot-bip44.mediawiki [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+A key derived with this derivation path pattern will be referred to as <tt>derived_key</tt> further
+in this document.
+
+===Address derivation===
+
+
+[[bip-0341.mediawiki#cite_ref-22-0|BIP 341]] states: "If the spending conditions do not require a
+script path, the output key should commit to an unspendable script path instead of having no
+script path. This can be achieved by computing theoutput key point as
```
Comment (junderw) [inline bip-taproot-bip44.mediawiki:58]:
```suggestion
script path. This can be achieved by computing the output key point as
```

Comment (achow101) [reply bip-taproot-bip44.mediawiki:58]: FIxed

**Comment by luke-jr** [comment]
Assigned BIP number 86

**Comment by benthecarman** [comment]
Is there a extended key version defined for BIP 86 like there was in BIP 84?

https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki#extended-key-version

**Comment by achow101** [comment]
> Is there a extended key version defined for BIP 86 like there was in BIP 84?

This intentionally does not define one.

**Comment by benthecarman** [comment]
> > Is there a extended key version defined for BIP 86 like there was in BIP 84?
> 
> This intentionally does not define one.

I think it'd be worthwhile adding the reasoning to the BIP

**Review by LeHoang88 — APPROVED** [review]

**Review by LeHoang88 — APPROVED** [review]

**Comment by Rspigler** [comment]
I've been away, so apologies for the post-merge comments.

I agree with @achow101 's comments on the ML - I believe this is mostly unnecessary with the use of descriptors, but as long as this is for single-sig only, ACK, as it can help with adoption.  Hopefully we will eventually move away from this however.

**Comment by benma** [comment]
Small post-merge comment: The test vectors contain addresses, so it would be great if the BIP mentioned that the addresses are bech32m (BIP-350).

**Pull request merged**